source_files/edge/rad_defs.h

```diff
@@ -279,7 +279,7 @@ typedef struct s_enabler_s
 	// script to enable/disable.  If script_name is NULL, then `tag' is
 	// the tag number to enable/disable.
 	char *script_name = nullptr;
-	int tag = 0;
+	uint32_t tag[2] = {0,0};
```

Comment:
Would it be clearer when indexing, instead of [0] and [1], maybe index by an enum or a defined index?  Something like: 

```
typedef enum
{
	tag_number = 0,
	tag_hash
}
tagtype_e;

or maybe 
```
#define TAG_NUMBER 0 
#define TAG_HASH 1 
```

Might be overthinking this one...

Reply:
This is a good idea as we expand RTS and COAL (or potentially Lua) interaction

Reply:
You might be. 
Though a code comment would be good i.e.
uint32_t tag[2] = {0,0};  //{TAG_NUMBER, TAG_HASH}